### ISSUE TITLE
Text inputs in Dacpac extension incorrectly announce invalid value in screen reader mode

### ISSUE THREAD

**Issue by ellbosch** [microsoft/azuredatastudio#7131]
Screen reader mode will incorrectly announce an invalid value in a very specific condition for text inputs. This bug exists for both Windows and Mac.

Steps to Reproduce:

1. Install the SQL Server Dacpac extension
2. Right click a database and select "Data-tier Application wizard"
3. Choose any of the 4 bullets
4. Enable screen-reader mode
5. Find a text input. If there isn't pre-populated text, input a random string
6. Clear the text input.
7. Type one character. Screen reader will announce an invalid character for the first typed character (even if valid), but will work correctly for subsequent keystrokes.

Expected functionality: screen reader should not announce an invalid value.

I expect that there is something wrong with validation in this file: src/sql/workbench/browser/modelComponents/inputbox.component.ts

**Comment by udeeshagautam** [comment]
@ellbosch, I tried this with latest bits and seem to speak correctly in following case. Please let me know if this was the scenario or something else you tried. I will close accordingly.

![image](https://user-images.githubusercontent.com/46980425/71048872-90678080-20f5-11ea-9a3b-6d62433b22f8.png)

**Comment by ellbosch** [comment]
Looks like it's fixed in the ADS official build.
